### ISSUE TITLE
Switch from self-describing JSON to hocon in configuration file

### ISSUE THREAD

**Issue by dilyand** [snowplow-incubator/snowplow-bigquery-loader#186]

**Comment by chuwy** [comment]
HOCON is an abbrebiatoin https://en.wikipedia.org/wiki/HOCON

**Comment by mabushey** [comment]
Yes, HOCON is an abomination, but it's better than a massive base64 json string in the CLI argument string. The other snowplow containers (collector, emitter) use HOCON config files. HOCON, JSON, I don't care, but lets not pass them on the CLI.
